pkg/apis/serving/k8s_validation_test.go

```diff
@@ -1509,7 +1509,7 @@ func TestContainerValidation(t *testing.T) {
 				ctx = config.ToContext(ctx, cfg)
 			}
 
-			got := ValidateContainer(ctx, test.c, test.volumes)
+			got := ValidateContainer(ctx, test.c, test.volumes, make(sets.String))
```

Comment:
probably worth a unit test for the new behaviour here too I think

Reply:
Yes we can cover more cases there by adding a test case field.